eth/ethconfig/config.go

```diff
@@ -137,8 +137,8 @@ type Config struct {
 
 	TxLookupLimit uint64 `toml:",omitempty"` // The maximum number of blocks from head whose tx indices are reserved.
 
-	// Whitelist of required block number -> hash values to accept
-	Whitelist map[uint64]common.Hash `toml:"-"`
+	// AllowList of required block number -> hash values to accept
+	AllowList map[uint64]common.Hash `toml:"-"`
```

Comment:
This change breaks compatibility with config files, so we cannot apply it without providing some kind of translator / deprecation mechanism. 

You can check how to do this in https://github.com/ethereum/go-ethereum/blob/master/cmd/geth/config.go#L238

Reply:
When I've done it, seeing the tag `"-"` I thought it was omitted, just here for reference not in the config.
A team member gave me this reference https://github.com/BurntSushi/toml/blob/ebe1404fc680a89e43605f720106b16be2ba141c/encode.go#L561-L563

Reply:
Oh, I didn't notice that! That's OK then.